clang/lib/CodeGen/CGCUDANV.cpp

```diff
@@ -905,10 +907,10 @@ llvm::Function *CGNVCUDARuntime::makeModuleCtorFunction() {
     GpuBinaryHandle = new llvm::GlobalVariable(
         TheModule, PtrTy, /*isConstant=*/false, Linkage,
         /*Initializer=*/
-        CudaGpuBinary ? llvm::ConstantPointerNull::get(PtrTy) : nullptr,
```

Comment:
this line should not be changed

Reply:
or change it to:

`RelocatableDeviceCode ? nullptr : llvm::ConstantPointerNull::get(PtrTy)`

Reply:
Yes, that one is good! Thanks.